### PR TITLE
added june meetup

### PR DESCRIPTION
**Pull request by igauravsehrawat**

### CODE REVIEW

**Comment by bravegnu** [comment]
Did you forget to add the June meetup info page?

**Comment by igauravsehrawat** [comment]
@bravegnu added.

File: content/meetup-june-2015.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Date: 2015-06-27 15:00
+Tags: Events
+
+<a style="float:right;" class="pure-button"
+href="http://www.meetup.com/Chennaipy/events/222732435/" target="_blank"><i class="fa
+fa-check-square-o"></i> RSVP on Meetup</a>
+
+## Date & Time
+
+  * 23th May
```
Comment (shrayasr) [inline content/meetup-june-2015.md]: Please change it to June

File: content/meetup-june-2015.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Location map:
+[http://www.google.com/maps?q=12.994337,80.247170](http://www.google.com/maps?q=12.994337,80.247170)
+
+## Schedule
+
+  * Lightning Talks (40 min)
+  * Networking Tea sponsored by [Clay Labs](http://claylabs.com/) (30 min)
+  * Lightning Talks (40 min)
+  * Discussions (20 min)
```
Comment (shrayasr) [inline content/meetup-june-2015.md]: This is no more the schedule. The meetup page should give you a better idea

**Comment by igauravsehrawat** [comment]
@shrayasr done

**Comment by shrayasr** [comment]
LGTM :+1:

**Comment by igauravsehrawat** [comment]
why did the build failed?

**Comment by shrayasr** [comment]
I didn't let it finish. Merged it before that

**Comment by igauravsehrawat** [comment]
ok. cool.

**Pull request merged**